### PR TITLE
[AKS] add C# config to readme.md

### PR DESCRIPTION
**Pull request by mboersma**
Adds a `--csharp` config for ACS/AKS/OSA. cc: @jskulavik.

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by nschonni** [comment]
Is `- repo: azure-sdk-for-net` needed in the `swagger-to-sdk:` list?

File: specification/containerservices/resource-manager/readme.md [hunk 1]
```diff
@@ -221,12 +219,40 @@ Please also specify `--csharp-sdks-folder=<path to "SDKs" directory of your azur
 csharp:
   azure-arm: true
   license-header: MICROSOFT_MIT_NO_VERSION
-  namespace: Microsoft.Azure.Management.ContainerServices
+  namespace: Microsoft.Azure.Management.ContainerService
   payload-flattening-threshold: 1
-  output-folder: $(csharp-sdks-folder)/ContainerServices/Management.ContainerServices/Generated
+  output-folder: $(csharp-sdks-folder)/ContainerService/Management.ContainerService/Generated
```
Comment (amarzavery) [inline specification/containerservices/resource-manager/readme.md]: @shahabhijeet - Is this change ok?

File: specification/containerservices/resource-manager/readme.md [hunk 2]
```diff
@@ -221,12 +219,40 @@ Please also specify `--csharp-sdks-folder=<path to "SDKs" directory of your azur
...
+  payload-flattening-threshold: 2
+  namespace: azure.mgmt.containerservice
+  package-name: azure-mgmt-containerservice
+  package-version: 1.0.0
+  clear-output-folder: true
+```
+``` yaml $(python) && $(python-mode) == 'update'
+python:
+  no-namespace-folders: true
+  output-folder: $(python-sdks-folder)/azure-mgmt-containerservice/azure/mgmt/containerservice
```
Comment (amarzavery) [inline specification/containerservices/resource-manager/readme.md]: Any reason to name the package containerServices (plural) in C# and containerservice (singular) in pythion?

**Review by amarzavery — COMMENTED** [review]
CIL

File: specification/containerservices/resource-manager/readme.md (hunk 2 above)
Comment (jskulavik) [reply specification/containerservices/resource-manager/readme.md]: They should both be singular as that is reflective of the ARM resource name.

**Comment by mboersma** [comment]
Let's revisit this after Azure/azure-sdk-for-python#3378 merges, because that will fix the multi-api CI failure we saw, so maybe changing that config isn't necessary.

**Comment by mboersma** [comment]
@amarzavery sorry for the thrashing--I think this is reviewable now. (It failed CI because the multi-api config hadn't landed in the Python SDK yet, but @lmazuel fixed that yesterday.)

**Review by amarzavery — APPROVED** [review]

**Pull request merged**